bcda/cli_test.go

```diff
@@ -223,26 +215,66 @@ func (s *CLITestSuite) TestImportCCLF8_InvalidMetadata() {
 	assert.NotNil(err)
 	assert.EqualError(err, "expected CCLF file number 8, but was 9")
 
-	metadata.cclfNum = 8
-	metadata.filePath = ""
+	metadata = cclfFileMetadata{}
 	err = importCCLF8(metadata)
 	assert.NotNil(err)
-	assert.EqualError(err, "file path must be provided")
+	assert.EqualError(err, "File CCLF8 not found")
 }
 
 func (s *CLITestSuite) TestImportCCLF9() {
 	assert := assert.New(s.T())
+	db := database.GetGORMDbConnection()
+	defer database.Close(db)
 
-	filePath := "../shared_files/cclf/T.A0001.ACO.ZC9Y18.D181120.T1000010"
-	metadata, err := getCCLFFileMetadata(filePath)
-	metadata.filePath = filePath
-	assert.Nil(err)
-	err = importCCLF9(metadata)
+	db.Unscoped().Delete(&models.CCLF9{})
+
+	acoID := "A0002"
+	fileTime, _ := time.Parse(time.RFC3339, "2018-11-20T10:00:00Z")
+	cclf9metadata := cclfFileMetadata{
+		env:       "test",
+		acoID:     acoID,
+		cclfNum:   9,
+		timestamp: fileTime,
+		filePath:  "../shared_files/cclf/T.A0001.ACO.ZC9Y18.D181120.T1000010",
+		perfYear:  18,
+		name:      "T.A0001.ACO.ZC9Y18.D181120.T1000010",
+	}
+
+	err := importCCLF9(cclf9metadata)
 	assert.Nil(err)
 
+	file := models.CCLFFile{}
+	db.First(&file, "aco_cms_id = ?", acoID)
```

Comment:
I know my CCLF8 tests searched by ACO ID too, but do you think it would be better to query on filename? Or maybe it doesn't matter because we're trying to confirm that this one file was imported, and that should be the only matching row in the table.

Reply:
changed both of ours to search by filename. I think this was a good find also